### PR TITLE
added game 2048

### PR DESCRIPTION
**Pull request by MartaChobaniuk**
[DEMO LINK](https://martachobaniuk.github.io/js_2048_game/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,175 @@
...
+function fillFieldCells() {
+  fillRandomCell();
+  fillGameField();
+}
+
+function handleArrows(e) {
+  switch (e.key) {
+    case 'ArrowLeft':
+      clickArrowLeft();
+      fillFieldCells();
```
Comment (etojeDenys) [inline src/scripts/main.js:139]: you can move this out of the switch and call only one time (after the switch)

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. styles here look broken
image

2. you don't need to add another cell, if we don't merge any other cells

https://github.com/mate-academy/js_2048_game/assets/70220667/d5244ec4-b9a8-4d5e-9986-43a26c65d5fa

**Review by etojeDenys — CHANGES_REQUESTED** [review]
image
still can reproduce this bug

**Comment by MartaChobaniuk** [comment]
I don't see this bug

**Review by lerastarynets — APPROVED** [review]
You've got this, keep pushing forward!

**Pull request closed without merging**